src/sig_stateful/lms/external/README

```diff
@@ -0,0 +1,7 @@
+This code attempts to be a usable implementation of the LMS Hash Based
+Signature Scheme from RFC 8554.
+
+See read.me for documentation how to use it.
+
+This is the ACVP branch - designed to be (optionally) compatible with the
+public ACVP server
```

Comment:
What server is this? Will we (eventually) (want to) be able to use it for testing?

Reply:
This is the NIST ACVP test server. NIST is in the process of adding LMS test vectors. An ACVP test harness would be required to handle this type of testing, which Cisco has an open source version. It would have to be integrated for OQS. 
This service is more targeted if FIPS algorithm testing is of interest.

Reply:
Test vector testing (known answer tests - KATs) should be part of CI testing as it already is for all other algorithms. If this can be done independently of a server, perfect. Can you add a pointer to this server(s documentation) such as to understand what its benefits would be?